examples/load_waveform_demo.ipynb

```diff
@@ -0,0 +1,507 @@
+      "text/plain": [
+       "<Figure size 1200x400 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# As of Feb 2023, the only time-domain eccentric model available through LAL is EccentricTD,\n",
+    "# which does not include the merger, and is restricted to nonspinning BHs.\n",
+    "# We expect more eccentric models will be added in the future.\n",
+    "kwargs = {\n",
```

Comment:
Here, also say:
Note that EccentricTD only allows ecc > 1e-5, and ignores the mean_ano parameter.